shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -542,67 +596,57 @@ async fn test_cassandra_request_throttling() {
 
     // this batch set should be allowed through
     {
-        let mut batch = Batch::new(BatchType::LOGGED);
+        let mut queries: Vec<String> = vec![];
         for i in 0..25 {
-            let statement = format!("INSERT INTO test_keyspace.my_table (id, lastname, firstname) VALUES ({}, 'text', 'text')", i);
-            batch.add_statement(&stmt!(statement.as_str())).unwrap();
+            queries.push(format!("INSERT INTO test_keyspace.my_table (id, lastname, firstname) VALUES ({}, 'text', 'text')", i));
         }
-        connection.execute_batch(&batch);
+        connection.execute_batch(queries);
     }
 
     std::thread::sleep(std::time::Duration::from_secs(1)); // sleep to reset the window
 
     // this batch set should not be allowed through
     {
-        let mut batch = Batch::new(BatchType::LOGGED);
+        let mut queries: Vec<String> = vec![];
         for i in 0..60 {
-            let statement = format!("INSERT INTO test_keyspace.my_table (id, lastname, firstname) VALUES ({}, 'text', 'text')", i);
-            batch.add_statement(&stmt!(statement.as_str())).unwrap();
+            queries.push(format!("INSERT INTO test_keyspace.my_table (id, lastname, firstname) VALUES ({}, 'text', 'text')", i));
         }
-        let result = connection.execute_batch_expect_err(&batch);
-        assert!(matches!(
+        let result = connection.execute_batch_expect_err(queries);
+        assert_eq!(
             result,
-            Error(
-                ErrorKind::CassErrorResult(cassandra_cpp::CassErrorCode::SERVER_OVERLOADED, ..),
-                ..
-            )
-        ));
+            CassandraError {
+                code: 0x1001.into(),
```

Comment:
wouldnt it make sense to just use the enum variant name considering we went to the trouble of setting it up?